catkin_ws/src/oh_translators/src/lcm2ros_ihmc.cpp

```diff
@@ -75,8 +76,8 @@ class LCM2ROS
   std::string chestLinkName_;
 
   // Parameters and Variables:
-  double default_transfer_time_;
-  double default_swing_time_;
+  double ihmc_transfer_time_;
```

Comment:
I believe these should be initialised as = 1.0 (as L155/156 were removed), otherwise when playing back logs from before using the new message or when not changing them after starting director the values set in L346/347 and L356/357 will be uninitialised memory. @mauricefallon should these be initialised here or handled below when the other message hasn't arrived?